app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/components/summary/ReimbursementsClaimsSummary.scala

```diff
@@ -53,13 +53,70 @@ object ReimbursementsClaimsSummary {
           )
         } ++ Seq(
         SummaryListRow(
-          key = Key(Text(messages(s"$key.total"))),
+          key = Key(Text(messages(s"$key.single.total"))),
           value = Value(Text(reimbursementClaims.map(_._2).sum.toPoundSterlingString))
         )
       )
     )
 
-  def multiple(
+  def singleForCYA(
+    reimbursementClaims: Map[TaxCode, BigDecimal],
+    key: String,
+    changeCallOpt: Option[Call]
+  )(implicit messages: Messages): SummaryList =
+    SummaryList(rows =
+      reimbursementClaims.toSeq
+        .map { case (taxCode, amount) =>
+          SummaryListRow(
+            key = Key(Text(messages(s"tax-code.${taxCode.value}"))),
+            value = Value(Text(amount.toPoundSterlingString)),
+            actions = changeCallOpt.map(changeCall =>
+              Actions(
+                items = Seq(
+                  ActionItem(
+                    href = changeCall.url,
+                    content = Text(messages("cya.change")),
+                    visuallyHiddenText = Some(messages(s"tax-code.${taxCode.value}"))
+                  )
+                )
+              )
+            )
+          )
+        } ++ Seq(
+        SummaryListRow(
+          key = Key(Text(messages(s"$key.total"))),
+          value = Value(Text(reimbursementClaims.values.sum.toPoundSterlingString)),
+          actions = changeCallOpt.map(changeCall =>
```

Comment:
Should we have a change link on the totals? I don't think it has them on the prototype.

Reply:
actually, there is a change link on the total row in the single journey, see https://cds-reimbursement-prototype.herokuapp.com/v7/C&E1179/single/check-your-answers?cmaeligible=true&second-document=true